frontend/src/pages/SocialPage/index.js

```diff
@@ -1,41 +1,59 @@
-import { COMPANY, WEBSITE, SOCIAL } from "constants/Subvisual.js";
+import { WEBSITE, SOCIAL } from "constants/Subvisual.js";
 
-import Header from "components/Header";
-import Body from "components/Typography/Body";
 import Logo from "components/Logo";
+import Heading2 from "components/Typography/Heading2";
+import Body from "components/Typography/Body";
 import SocialMediaLink from "components/SocialMediaLink";
 
+import { ReactComponent as WheelImageTopRight } from "assets/svgs/wheel-background/top-right.svg";
+import { ReactComponent as WheelImageBottom } from "assets/svgs/wheel-background/bottom.svg";
+
 import styles from "./index.module.css";
 
 const SocialPage = () => {
   return (
-    <>
-      <Header>{COMPANY}</Header>
-      <Logo>Thanks for playing!</Logo>
-
-      <div className={styles.subtitle}>
-        <Body>
-          Find us at{" "}
-          <a href={WEBSITE} target="_blank" rel="noreferrer">
-            {WEBSITE.substring(8)}
-          </a>{" "}
-          and on our social media:
-        </Body>
+    <div className={styles.root}>
+      <div className={styles.svgWrapperTop}>
+        <WheelImageTopRight />
       </div>
 
-      <ul className={styles.links}>
-        {Object.keys(SOCIAL).map((channel) => {
-          return (
-            <li className={styles.link} key={SOCIAL[channel]}>
-              <SocialMediaLink
-                url={SOCIAL[channel].url}
-                logo={SOCIAL[channel].logo}
-              />
-            </li>
-          );
-        })}
-      </ul>
-    </>
+      <div className={styles.main}>
+        <div className={styles.logoWrapper}>
+          <Logo />
+        </div>
+
+        <div className={styles.headingWrapper}>
+          <Heading2>Thanks for playing!</Heading2>
```

Comment:
I believe we decided to only use `margin-bottom` for spacing. But `Heading2` has also `margin-top`. Once you remove it, you might need to readjust.